lib/calendar_builder.rb

```diff
@@ -0,0 +1,43 @@
+class CalendarBuilder < Struct.new(:view, :date, :callback)
+    HEADER = %w[Sunday Monday Tuesday Wednesday Thursday Friday Saturday]
```

Comment:
Going to leave this one alone for now in interest of time/since the days won't ever change